src/config/mod.rs

```diff
@@ -215,6 +215,7 @@ impl Config {
             let version = env!("CARGO_PKG_VERSION");
             let required_version = self.required_version();
             if version != required_version {
+                // FIXME: when format args are inlined, this line generates error[internal]
```

Comment:
I wouldn't want to add a fixme as part of this. Would be better to just leave this as is, or manually multiline the string so that it can fit

Reply:
@calebcartwright thx, fixed